packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -172,6 +172,16 @@
       line-height: 1.5rem;
     }
 
+    @media (min-width: 600px) {
```

Comment:
The rest of this file uses `max-width: $mdc-toolbar-mobile-breakpoint`. It should stay consistent. 
I think it should be moved to the title class and replace line 172
```
@media (max-width: $mdc-toolbar-mobile-breakpoint) { 
  line-height: 1.5rem;
}
```

Reply:
True, I did consider using `$mdc-toolbar-mobile-breakpoint`, but that's not mobile first. So my question would then be, why is the entire component not developed mobile first?

Setting a line-height override using `max-width: $mdc-toolbar-mobile-breakpoint` would mean that it would use the `@include mdc-typography(display1);` settings for all screen sizes above `$mdc-toolbar-mobile-breakpoint`. This would mean it will take 2.5rem (https://github.com/material-components/material-components-web/blob/master/packages/mdc-typography/_variables.scss#L62) which is also not what we want it to be.
Besides, I think we only need the override specifically for the `.mdc-toolbar--fixed-lastrow-only` variant right?

Reply:
While I agree that this should be mobile first, this component isn't currently built that way, so I don't see any use in having one code block be different from the rest. 

This behavior also affects the `mdc-toolbar--flexible` variant. 

2.5rem gives the desired behavior in all screen sizes except for mobile, which is fixed by adjusting the line height to 1.5. The only issue remaining is the icon vertical alignment issue, which will be addressed by #2138

Reply:
@williamernest should be updated.